packages/ses/test/test-make-evaluator.js

```diff
@@ -0,0 +1,109 @@
+import '../index.js';
```

Comment:
I did not review this file but assume it’s a mechanically identical to the former `test-evaluate.js`. Please note anywhere it’s not just mechanically replacing `performEval` with a curried `safeEvaluate`.

Reply:
I honestly do not remember after 2 months, but I'll double check. I do believe it was purely mechanical (and formatting).

Reply:
I have now split this change into 3 commits:
- Split the tests into their respective areas. Previously some tests were mixing concerns (sloppy mode, endowments)
- Mechanically prepare test file for smaller diff with next commit
- Minimal update from the actual refactor

This was rebased, and is this better reviewed commit by commit.